ONSdigital/eq-author
# Fix logging for elastic beanstalk

### Changes
In pre-production one of the instances was unable to write the log files do to permissions issue on the box. This causes the application to fall over on startup. It seems to be a common problem with django and elastic beanstalk. The recommend approach seems to be to create a directory in /var/log on the ec2 instance and give it the correct permission for the application, then all application log files should be written there.

### How to test
1. Deploy this change to elastic beanstalk
2. Check both instances are responding correctly
3. Check the log file exists in /var/log/author

### Who can test
Anyone apart from @warrenbailey
